stew/base36.nim

```diff
@@ -0,0 +1,222 @@
+## Copyright (c) 2025 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+## This module implements Base36 encoding and decoding procedures.
+## This module supports two Base36 alphabets: lowercase `Base36Lc` and uppercase `Base36Uc`.
+
+## This module follows Base36 as defined in https://github.com/multiformats/multibase/blob/f378d3427fe125057facdbac936c4215cc777920/rfcs/Base36.md
```

Comment:
https://status-im.github.io/nim-style-guide/formatting.style.html#style-formattingstyle
> We strive to follow [NEP-1](https://nim-lang.org/docs/nep1.html) for style matters - naming, capitalization, 80-character limit etc.

Obviously, this doesn't mean breaking long URLs into multiple pieces; some lexical units are atomic. And in practice, we don't always use strict 80-character lines; "strive" there does point to the non-absoluteness of this guideline. But this one is a bit long without putting the URL on a separate line.